derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -302,8 +333,30 @@ impl Field {
         errors.finish()
     }
 
-    /// Populate `self.field_attrs` and `self.setter_attrs` by draining `self.attrs`
-    fn unnest_attrs(mut self) -> darling::Result<Self> {
+    /// Resolve and check (post-parsing) options which come from multiple darling options
+    ///
+    ///  * Check that we don't have a custom field builder *and* a default value
+    ///  * Populate `self.field_attrs` and `self.setter_attrs` by draining `self.attrs`
+    fn resolve(mut self) -> darling::Result<Self> {
+        // `field.build` is stronger than `default`, as it contains both instructions on how to
+        // deal with a missing value and conversions to do on the value during target type
+        // construction. Because default will not be used, we disallow it.
+        if let Field {
+            default: Some(field_default),
+            field:
+                FieldLevelFieldMeta {
+                    build: Some(_custom_build),
+                    ..
+                },
+            ..
+        } = &self
+        {
+            return Err(darling::Error::unsupported_format(
+                r#"#[builder(default)] and #[builder(build="...")] cannot be used together"#,
+            )
+            .with_span(field_default));
```

Comment:
Please use `Error::accumulator()` here to avoid losing errors from `distribute_and_unnest_attrs` below.